_posts/2017/2017-10-05-codetalks-2017-back-on-the-big-screen.md

```diff
@@ -0,0 +1,85 @@
+As a result code.talks had to shrink to 1.700 people from 1.900 last year, which is still 200 people more than two years ago.
+
+And yes - again it proved that a cinema is a pretty neat location for a software development conference.
+It seems that cosy seats, popcorn, and nachos are things that most of the attendees seemed comfortable with.
+
+Here are our highlights of this year's code.talks edition.
+
+## Highlights
+
+### 10 Tips for failing at microservices
+
+Two years ago microservices was one of the main topics of the code.talks.
+Everybody talked about it but very few actually had production experience.
+Now the microservices hype cycle passed the "Peak of Inflated Expectations" and we are seeing a more realistic picture.
+
+[David Schmitz (@koenighotze)](https://twitter.com/koenighotze) chimed in here and provided us with a very sarcastic talk about "10 Tips for failing at microservices".
+It was great infotainment.
+In his talk David touched many topics that are really hard to get right in a microservices context like polyglot services, build automation, shared code, domain modeling, and team collaboration and told us how to mess up.
+
+<blockquote class="twitter-tweet" data-lang="de"><p lang="en" dir="ltr">Hey <a href="https://twitter.com/hashtag/codetalkshh?src=hash&amp;ref_src=twsrc%5Etfw">#codetalkshh</a>, you can find my slides about <a href="https://twitter.com/hashtag/failing?src=hash&amp;ref_src=twsrc%5Etfw">#failing</a> at <a href="https://twitter.com/hashtag/microservices?src=hash&amp;ref_src=twsrc%5Etfw">#microservices</a> here: <a href="https://t.co/gA07TMWHYR">https://t.co/gA07TMWHYR</a> Really awesome audience.</p>&mdash; 👑Der Schmitz💾 (@koenighotze) <a href="https://twitter.com/koenighotze/status/913357102613237760?ref_src=twsrc%5Etfw">28. September 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+If you liked David Schmitz' talk you might think about seeing him again at the JUG Hamburg giving a talk about [painless object functional Java with Vavr](https://www.meetup.com/de-DE/jug-hamburg/events/243300696/).
+
+### An introduction to Kotlin by example
+
+During the second day of this years code.talks [Dmitry Kandalov](https://twitter.com/dmitrykandalov?lang=en) gave a very practical introduction to [JetBrains](https://www.jetbrains.com/) own programming language [Kotlin](https://kotlinlang.org/), which just recently was [announced as an official Android language](https://blog.jetbrains.com/kotlin/2017/05/kotlin-on-android-now-official/).
+With basic live-coding examples, he demonstrated how simple Kotlin can be integrated within existing JVM based projects and how easy it is to use libraries from the Java ecosystem in Kotlin which rapidly speeds up development.
+
+_"Kotlin is what Java 9 should have been if it as written today from scratch."_ - This quote from the talk description pretty much summarized the tone of Kandalov's talk.
```

Comment:
Kotlin is what Java 9 should have been if it was written... the talk description pretty much summarizes the tone of...